packages/blocks/src/__internal__/rich-text/shortcuts.ts

```diff
@@ -278,6 +278,7 @@ export class Shortcuts {
           return PREVENT_DEFAULT;
         }
         const page = getDefaultPageBlock(model).page;
```

Comment:
This looks a bit weird here:

1. I think `getDefaultPageBlock` should return `page` rather than `{ page: Page }`.
2. Why not use `model.page`?

Reply:
1. `getDefaultPageBlock()` returns the `DefaultPageBlockComponent` rather than `Page`, which is a property of `DefaultPageBlockComponent`.
2. YES!!!